.github/workflows/cicd-release-validation.yml

```diff
@@ -242,12 +242,44 @@ jobs:
       gather_dumps: true
 
   # Run the quick stress tests in GitHub.
-  stress:
+  # Stress Tests should run 3x the normal CI/CD workflow
```

Comment:
This looks like adding the same stress tests that @Alan-Jowett has removed in his PR to deprecate ebpf_performance.exe.   Please discuss is this a valid stress tests to run?

Reply:
Thanks, I'll remove it from here as well.